src/vs/workbench/services/editor/common/editorResolverService.ts

```diff
@@ -59,6 +59,21 @@ export interface IEditorType {
 configurationRegistry.registerConfiguration(editorAssociationsConfigurationNode);
 //#endregion
 
+//#region defaultBinaryEditor
+export const defaultBinaryEditorSettingId = 'workbench.defaultBinaryEditor';
+
+const defaultBinaryEditorConfigurationNode: IConfigurationNode = {
+	...workbenchConfigurationNodeBase,
+	properties: {
+		'workbench.defaultBinaryEditor': {
+			type: 'string',
```

Comment:
Would be nice if we could offer the user a list of valid binary editors. We do the same for the `workbench.editor.autoLockGroups` setting, not sure it would be possible here as well?

Reply:
Moved the setting to the same place `workbench.editor.autoLockGroups` is and just followed that setup. Let me know if it shouldn't go there.